packages/jsii-pacmak/test/targets/version-utils.test.ts

```diff
@@ -63,6 +63,18 @@ const examples: Record<
     nuget: '[,0.1.2]',
     python: '<=0.1.2',
   },
+
+  // Somewhat unusual ranges
+  '*': {
+    maven: '[0.0.0,]',
+    nuget: '[0.0.0,]',
+    python: '>=0.0.0',
+  },
+  '1.2.*': {
+    maven: '[1.2.0,1.3.0)',
+    nuget: '[1.2.0,1.3.0)',
+    python: '>=1.2.0, <1.3.0',
+  },
```

Comment:
Can you also add one for `1.*`? (I reckon this would be a valid way to denote major-pinned, isn't it?)